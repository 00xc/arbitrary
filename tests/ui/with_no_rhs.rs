use arbitrary::Arbitrary;

#[derive(Arbitrary)]
struct Point {
    #[arbitrary(with)]
    x: i32,
    y: i32
}

fn main() { }
