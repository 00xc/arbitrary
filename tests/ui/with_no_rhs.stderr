error: Invalid syntax for #[arbitrary], `with` is missing assignment.
 --> tests/ui/with_no_rhs.rs:5:16
  |
5 |     #[arbitrary(with)]
  |                ^^^^^^
